CHANGELOG.next.toml

```diff
@@ -1,24 +1,5 @@
-# Example changelog entries
-# [[aws-sdk-rust]]
-# message = "Fix typos in module documentation for generated crates"
-# references = ["smithy-rs#920"]
-# meta = { "breaking" = false, "tada" = false, "bug" = false }
-# author = "rcoh"
-#
-# [[smithy-rs]]
-# message = "Fix typos in module documentation for generated crates"
-# references = ["smithy-rs#920"]
-# meta = { "breaking" = false, "tada" = false, "bug" = false }
-# author = "rcoh"
-
-[[aws-sdk-rust]]
-message = "Convert several `info` spans to `debug` in aws-config"
-references = ["smithy-rs#1087"]
-meta = { "breaking" = false, "tada" = false, "bug" = false }
-author = "rcoh"
-
 [[smithy-rs]]
-message = "Improve docs on `Endpoint::{mutable, immutable}`"
-references = ["smithy-rs#1087"]
+message = "Provide SigningService creation via owned String"
```

Comment:
In the changelog, keep the entries that were here already and just add your new entry.

Reply:
Shoot. Will fix that, thanks.

Reply:
Hopefully new push was correctly picked up. Marking as resolved.

Reply:
Looks like some new entries snuck in during the time in between 😅 Needs another fix